docs/README.md

```diff
@@ -28,6 +28,8 @@ wmcb run --ignition-file $IGNITION_FILE_PATH --kubelet-path $KUBELET_PATH
 
 ## Testing
 
+### Bootstrapper
```

Comment:
I am not fan of using bootstrapper. I prefer `Windows Machine Config Bootstrapper`

Reply:
Changed